src/org/parosproxy/paros/view/SiteMapPanel.java

```diff
@@ -390,6 +391,19 @@ public void reloadContextTree() {
 		this.contextTree.nodeStructureChanged(root);
 	}
 
+	public Context getSelectedContext() {

```

Comment:
Worth adding a since JavaDoc tag (and document it a little, that might return null if nothing selected or not selected a context node).